### PR TITLE
[PLATFORM-1170]: Fix exclude options

### PR DESCRIPTION
**Pull request by MaeIsBad**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-1170

### CODE REVIEW

File: lib/prima_opentelemetry_ex/instrumentation/optional.ex [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    Creates a maybe_setup function that instruments a library using an optional instrumentation library.
+    
+    Executes the `do` block if:
+      * `to_instrument` is available
+      * `instrumenting_library` is available
+      * `feature_name` is not in the prima_opentelemetry_ex exclude list
+    
+    * `to_instrument`: the module that will be instrumented(eg. `Ecto`)
+    * `instrumenting_library`: the module that will be used to instrument(eg. `OpentelemetryEcto`)
+    * `feature_name`: atom specifing the name of the feature. Typically the name of the to_instrument library in atom format(eg. `:ecto`)
```
Comment (cpiemontese) [inline lib/prima_opentelemetry_ex/instrumentation/optional.ex:14]:
```suggestion
    * `feature_name`: atom specifying the name of the feature. Typically the name of the to_instrument library in atom format(eg. `:ecto`)
```

**Review by cpiemontese — COMMENTED** [review]
I've become a spell checker :robot: 

Jokes aside I really like the `Optional.create` macro. I wonder about the naming, I was thinking if something like the following would be possibile

```elixir
Optionally.instrument Ecto,
  with: OpentelemetryEcto,
  feature: :ecto
```

But I think this may enter the bike-shedding realm

**Comment by MaeIsBad** [comment]
I like your API a lot more, changed

**Review by cpiemontese — APPROVED** [review]
Nice :+1:

**Pull request merged**